cloudsync/youtube.py

```diff
@@ -15,6 +15,9 @@
 
 log = logging.getLogger(__name__)
 
+# Quota errors may contain either one of the following
+QUOTA_ERROR = 'dailyLimitExceeded'
```

Comment:
SPECIFICITY
This may be clearer as 'API_QUOTA_ERROR_MSG', to indicate that:
1. It's the API quota (It thought this was the upload quota on my first read).
2. It's a string and not an `Exception`.

Reply:
✅